openTSNE/nearest_neighbors.py

```diff
@@ -398,7 +389,8 @@ def build(self, data, k):
             n_trees=n_trees,
             n_iters=n_iters,
             max_candidates=60,
-            n_jobs=n_jobs_pynndescent,
+            n_jobs=self.n_jobs,
+            verbose=1 if self.verbose>1 else 0,
```

Comment:
A bit nit-picky, but could you add spaces around the `>`, so `verbose=1 if self.verbose > 1 else 0`, or better yet `verobose=self.verbose > 1`.

Reply:
Yes, that's a better way to write it, thanks. Done.